.github/workflows/trigger_libraries.yml

```diff
@@ -0,0 +1,24 @@
+name: Trigger Libraries
+
+on:
+  push:
+    branches: gh-pages
+    paths: 'package_adafruit_index.json'
```

Comment:
this workflow only run when there is changes to index json on `gh-pages`,  when it run it will send `repository_dispatch` event to list of Arduino libraries. Those library workflow need to add `repository_dispatch` to event list in addition to push, pull_request e.g https://github.com/adafruit/Adafruit_TinyUSB_Arduino/blob/master/.github/workflows/githubci.yml#L3